department-of-veterans-affairs/va.gov-cms
# VACMS-19377: Fixed for tabbing on menu systems.

## Description

Closes #19377 

## Testing done


## Screenshots


## QA steps

When logged in to the Drupal backend, start to tab through the main menus with your keyboard (content, structure, etc).
The tabbing should work on the main menus but require you to hit enter to tab in to submenus.

As user _uid_ with _user_role_
1. Tab through Drupal back end menus
2. Hit enter to enter a sub menu.
3. If you do not hit enter you should remain in the top level menu when tabbing

### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
